developers/weaviate/api/graphql/additional-operators.md

```diff
@@ -32,9 +34,11 @@ import GraphQLFiltersExample from '/_includes/code/graphql.filters.example.mdx';
 
 ## Limit argument
 
-Supported by the `Get{}`, `Explore{}` and `Aggregate{}` function.
+The `limit` argument restricts the number of results. These operators support `limit`:
```

Comment:
Hmm. I thought we called these these "functions" rather than operators. What do we call them elsewhere?